electron/build/im.vector.Riot.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Riot
+Comment=A feature-rich client for Matrix.org
+Exec=riot-web
+Terminal=false
+Type=Application
+Icon=im.vector.Riot
+StartupWMClass="Riot"
+Categories=Network;InstantMessaging;Chat;IRCClient
```

Comment:
This doesn't match the cats being used in the Linux app, (though Chat should be in them and I missed it)
i.e remove IRCClient